.NET/Microsoft.Recognizers.Text.DateTime/Chinese/Parsers/DatePeriodParserChs.cs

```diff
@@ -1131,6 +1136,108 @@ private DateTimeResolutionResult ParseQuarter(string text, DateObject referenceD
             return ret;
         }
 
+        private DateTimeResolutionResult ParseDecade(string text, DateObject referenceDate)
+        {
+            var ret = new DateTimeResolutionResult();
+            int century = referenceDate.Year / 100;
+            int decade;
+            int beginYear;
+            int beginMonth = 1, endMonth = 12;
+            int beginDay = 1, endDay = 31;

```

Comment:
Why do you need month and day to parse decade?

Reply:
I use these numbers later when specifying the first day of a year and the last day of the year, like 1970s is 1970-01-01 to 1979-12-31.